src/main/java/edu/cornell/kfs/concur/rest/authentication/ConcurAuthenticationFilter.java

```diff
@@ -47,9 +50,28 @@ public void doFilter(ServletRequest servletRequest, ServletResponse response, Fi
                 return;
             }
         }
-        LOG.debug("doFilter(): Concur authentication was successful");
+        LOG.info("doFilter(): Concur authentication was successful");
         filterChain.doFilter(servletRequest, response);
     }
+    
+    private void logRequst(HttpServletRequest request) {
```

Comment:
code looks good, just has a spelling error in method name and where the method name is used in logging.